projects/angular/src/layout/nav/_links.clarity.scss

```diff
@@ -4,11 +4,37 @@
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 @use '../../utils/mixins';
-@use '../../utils/variables/variables.global';
+@use '../../utils/variables/variables';
 @use 'variables.nav' as nav-variables;
 
 @include mixins.exports('links.clarity') {
   // only imported for demos. gives a static view of links in various states.
+  a:link {
+    @include mixins.css-var(
+      font-weight,
+      clr-link-font-weight,
+      variables.$clr-p1-font-weight,
+      variables.$clr-use-custom-properties
+    );
+    @include mixins.css-var(
+      font-size,
+      clr-link-font-size,
+      variables.$clr-p1-font-size,
+      variables.$clr-use-custom-properties
+    );
+    @include mixins.css-var(
+      line-height,
+      clr-link-line-height,
+      variables.$clr-p1-line-height,
+      variables.$clr-use-custom-properties
+    );
+    @include mixins.css-var(
+      letter-spacing,
+      clr-link-letter-spacing,
+      variables.$clr-p1-letter-spacing,
+      variables.$clr-use-custom-properties
+    );
+  }
```

Comment:
Just make sure we need those link changes. Typography and Buttons design specifications are contradicting.

Reply:
Discussed offline and decided to use `inherit` as default which will result in the same values but keep the exposed properties.